### DISCUSSION TITLE
Substance/material subclass?

### DISCUSSION THREAD

**Discussion by dlutz2** [CommonCoreOntology/CommonCoreOntologies#394] [Announcements]
Are there plans in a future version of CCO to include a subclass of material entity for substances/materials?
We are mapping in a legacy coding scheme which has large number of elements of the pattern:
XX_facility is a cco:Facility that extracts/processes/refines/stores/consumes ZZ_Material

It would be convenient to have a defined class to hook our extensive lists of raw materials, chemicals etc.

  thanks

**Comment by neilotte** [comment]
@mark-jensen @APCox  -- Are there previous discussions around portions of material within the last few years that you are tracking? I agree with @dlutz2 that this would be helpful (@johnbeve  FYSA)

**Comment by neilotte** [comment]
@dlutz2 Following your original request, CCO moved many of its artifact classes under material entity and out from under object. This doesn't fully address your original inquiry, which warrants further discussion.
